File: Web/Repositories/Models/RepositoryModel.cs
namespace Repositories.Models;

public class RepositoryModel
{
    public Guid Id { get; set; }

    public IndexJsonModel Index { get; set; }

    public string IndexMd { get; set; }

    public dynamic Instructions { get; set; }

    public string License { get; set; }

    public string CodeOfConduct { get; set; }
}

File: Web/krk-web/src/app/user/pages/recipe-page/responses/repository.model.ts
import {IndexModel} from '../models/index.model';
import {InstructionsModel} from '../models/instructions.model';
import {BaseModel} from '@shared/models/base/base.model';

export interface RepositoryModel extends BaseModel {
  index: IndexModel;
  indexMd: string;
  instructions: InstructionsModel;
  License: string;
  codeOfConduct: string;
}
